### PR TITLE
chore: deployment is 3-3-1 replicas

### PR DESCRIPTION
**Pull request by lzap**
SRE bot informs us about a check, we need to turn it off for statuser.

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
Potentionaly we could try adding annotation to the given deployment (statuser), it allows annotations, but hard to say if it would work adding this specific annotation just for the given deployment.

File: deploy/clowdapp.yaml [hunk 1]
```diff
@@ -104,7 +106,7 @@ objects:
                 cpu: ${CPU_REQUESTS}
                 memory: ${MEMORY_REQUESTS}
         - name: statuser
-          # statuser pod must have exactly one replicas
+          # statuser pod is just one replica for easier throttling
           replicas: 1
```
Comment (ezr-ondrej) [inline deploy/clowdapp.yaml:110]: I'd love to try the annotations here, but up to you :)
```suggestion
          replicas: 1
          metadata:
            annotations:
              ignore-check.kube-linter.io/minimum-three-replicas: "statuser pod runs in a single instance"
```

Comment (lzap) [reply deploy/clowdapp.yaml:110]: Let’s try: https://github.com/RHEnVision/provisioning-backend/pull/467

Let’s merge whatever works.

**Comment by lzap** [comment]
I am going to merge this because stage is stuck and Ondra is offline.

**Comment by lzap** [comment]
I am merging back to 2-2-1 because we were hitting CPU limits. We will disable the linter to stop notifications about 3 minimum pods.

**Pull request merged**